client-js/app/elements/labrad-registry.html

```diff
@@ -2,6 +2,7 @@
 <link rel="import" href="../bower_components/paper-toast/paper-toast.html">
 <link rel="import" href="app-link.html">
 <link rel="import" href="dir-item.html">
+<link href='https://fonts.googleapis.com/css?family=Roboto+Mono' rel='stylesheet' type='text/css'>
```

Comment:
just curious, if this is included in another element as well, will it get deduped and downloaded only once?